packages/testharness/src/index.js

```diff
@@ -1,6 +1,14 @@
+import 'script-loader!../node_modules/@babel/standalone/babel.min.js';
```

Comment:
I know it's ugly, but it reduce boilerplate code and this is for test harness only.